react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js

```diff
@@ -97,11 +99,31 @@ export const EventTester = () => {
   )
 
   const openDrillMenuClick = useCallback(
-    (event) => {
-      // TODO links data needs to be populated
-      tileSDK.openDrillMenu({ links: [] }, event)
+    (_event) => {
+      let event = { pageX: 0, pageY: 0 }
+      let links = []
+      if (visualizationData?.queryResponse?.data) {
```

Comment:
TODO - add utility methods to the extension SDK
1. get all drill links for column name - returns an array of drill links for the specified column name
2. 1. get drill links for column name and row index - returns drill links for the specified column name and row index